### PR TITLE
Fix IPv6 ansible and registration tests

### PR DESCRIPTION
**Pull request by shubhamsg199**
Fix failing IPv6 ansible and registration tests.
The test is failing because the custom repos are created with download.devel url to download packages which can't be reached over IPv6

### CODE REVIEW

File: tests/foreman/cli/test_report.py [hunk 1]
```diff
@@ -137,6 +139,7 @@ def test_positive_run_puppet_agent_generate_report_when_no_message(
     :parametrized: yes
     """
     sat = session_puppet_enabled_sat
+
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_report.py:142]: Do we need proxy configuration here for configure_puppet?

File: tests/foreman/ui/test_registration.py [hunk 2]
```diff
@@ -204,6 +206,7 @@ def test_positive_global_registration_end_to_end(
     ]
     for pair in expected_pairs:
         assert pair in cmd
+
```
Comment (Gauravtalreja1) [inline tests/foreman/ui/test_registration.py:209]: Do we need proxy configuration here for installing packages?

Comment (shubhamsg199) [reply tests/foreman/ui/test_registration.py:209]: Yes, it creates custom repos in this test using `settings.repos[f'rhel{rhelver}_os']`

Comment (Gauravtalreja1) [reply tests/foreman/ui/test_registration.py:209]: Could you add proxy configure step here?

**Pull request merged**